integrationtest/showcase/library/src/test/java/org/hibernate/search/integrationtest/showcase/library/functional/LibraryFunctionalIT.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.integrationtest.showcase.library.functional;
+
+import org.hibernate.search.integrationtest.showcase.library.TestActiveProfilesResolver;
+import org.hibernate.search.integrationtest.showcase.library.model.Book;
+import org.hibernate.search.integrationtest.showcase.library.model.BookMedium;
+import org.hibernate.search.integrationtest.showcase.library.model.Library;
+import org.hibernate.search.integrationtest.showcase.library.model.LibraryServiceOption;
+import org.hibernate.search.util.impl.test.annotation.TestForIssue;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@TestPropertySource(properties = { "automatic_indexing.strategy=session" })
+@ActiveProfiles(resolver = TestActiveProfilesResolver.class)
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+public class LibraryFunctionalIT {
```

Comment:
This seem to imply other tests are not functional, which is wrong. Maybe `LibraryBatchInsertIT`?